### ISSUE TITLE
[KEY-260] A11y check and remidate

### ISSUE THREAD

**Issue by perkinsjr** [unkeyed/unkey#566]
Go through our entire site check for A11y violations, and resolve them. 

<sub>From [SyncLinear.com](https://synclinear.com) | [KEY-260](https://linear.app/unkey/issue/KEY-260/a11y-check-and-remidate)</sub>
